vault_core/src/vault.rs

```diff
@@ -1331,8 +1344,9 @@ mod tests {
             std::mem::size_of::<PodU16>() + // reward_fee_bps
             std::mem::size_of::<PodU16>() + // program_fee_bps
             std::mem::size_of::<PodBool>() + // is_paused
+            std::mem::size_of::<PodU64>() + // last_start_state_update_slot
             1 + // bump
-            261; // reserved
+            253; // reserved
```

Comment:
missing a field? this is 251 above

Reply:
fixing

Reply:
next_withdrawal_fee_bps was missing ( and is missing from master branch as well )